src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/FunctionalQueueConfig.java

```diff
@@ -17,19 +22,41 @@
     @Value("${queue.envelopes.write-connection-string}")
     private String queueWriteConnectionString;
 
+    @Value("${queue.envelopes.read-connection-string}")
+    private String queueReadConnectionString;
+
     @Bean
-    QueueClient testWriteClient() throws ServiceBusException, InterruptedException {
+    public QueueClient testWriteClient() throws ServiceBusException, InterruptedException {
         return new QueueClient(
             new ConnectionStringBuilder(queueWriteConnectionString),
             ReceiveMode.PEEKLOCK
         );
     }
 
+    @Bean(name = "dlqReceiver")
+    public Supplier<IMessageReceiver> dlqReceiverProvider() {
+        return () -> {
+            try {
+                return ClientFactory.createMessageReceiverFromConnectionStringBuilder(
+                    new ConnectionStringBuilder(StringUtils.join(queueReadConnectionString, "/$deadletterqueue")),
+                    ReceiveMode.PEEKLOCK
+                );
+            } catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
```

Comment:
i know it's just a test config but should the exception be thrown here as well?

Reply:
I'll create a new pr. As discussed, it should be thrown in the main code as well